### PR TITLE
Add a link back the Github page to address #1052

### PR DESCRIPTION
**Pull request by scottchiefbaker**
Add a simple link to the man page to report bugs on Github.

### CODE REVIEW

**Comment by scottchiefbaker** [comment]
It's hard to tell, but it looks like the `fd --help` page is generated from the man page? I think it would be a good idea to have something on the `--help` page also.

**Comment by tmccombs** [comment]
> It's hard to tell, but it looks like the fd --help page is generated from the man page?

No, the `--help` output is generated by clap from the options defined in src/app.rs

**Review by tmccombs — APPROVED** [review]

**Comment by scottchiefbaker** [comment]
Thanks for the clarification. I have updated the PR to include changes to `src/app.rs` to include the Github URL.

File: src/app.rs [hunk 1]
```diff
@@ -12,9 +12,9 @@ pub fn build_app() -> Command<'static> {
         .color(clap_color_choice)
         .setting(AppSettings::DeriveDisplayOrder)
         .dont_collapse_args_in_usage(true)
-        .after_help(
-            "Note: `fd -h` prints a short and concise overview while `fd --help` gives all \
-                 details.",
+        .after_help("Bugs can be reported on Github: https://github.com/sharkdp/fd\n
```
Comment (tmccombs) [inline src/app.rs]: I think you might need a `\` at the end of this line

Comment (scottchiefbaker) [reply src/app.rs]: This line was updated (simplified) in a subsequent push.

**Comment by scottchiefbaker** [comment]
Any updates on this? Anything else I can fix/update.

**Review by tmccombs — APPROVED** [review]

File: doc/fd.1 [hunk 2]
```diff
@@ -455,5 +455,7 @@ $ fd -e py
 .TP
 .RI "Open all search results with vim:"
 $ fd pattern -X vim
+.SH BUGS
+Bugs can be reported on Github: https://github.com/sharkdp/fd
```
Comment (tavianator) [inline doc/fd.1:459]: If we're talking about bugs, should we link directly to https://github.com/sharkdp/fd/issues?

**Comment by scottchiefbaker** [comment]
What's next for this PR? I'd really love to see this get merged.

**Comment by sharkdp** [comment]
Thank you! I added `/issues`, as @tavianator suggested and added a CHANGELOG entry. I also removed the comment from the help text because I would really like to keep the `fd -h` output small and concise. If there is a way to add it to `fd --help` only, I'm open to adding it there.

**Review by sharkdp — APPROVED** [review]

**Comment by scottchiefbaker** [comment]
@sharkdp I'd really like to see a link in the `--help` output. It's entirely possible someone will have the binary and not the man page and have no way to get back to the project.

How do I put text in the `--help` output and not in the `-h` output?

**Comment by tavianator** [comment]
@scottchiefbaker Clap seems to have an `after_long_help()` method: https://docs.rs/clap/latest/clap/builder/struct.App.html#method.after_long_help

**Comment by scottchiefbaker** [comment]
@tavianator good find! I submitted a PR with some `after_long_help()`.

**Pull request merged**